Data/Boundaries/Smoothing/Motu_Bay1_v22p5.txt

```diff
@@ -0,0 +1,187 @@
+177.61612328 -37.82127620

```

Comment:
I think the main issue is this file should now be named "Omaio_Bay1_v22p5.txt" since Cameron changed the "main" basin to be Omaio_Bay1_v22p5 now instead of Motu_Bay1_v22p5.